### PR TITLE
settings: add jinja2 support (bug 1873071)

### PR DESCRIPTION
**Pull request by zzzeid**
- add jinja2 to requirements
- update settings to pick up jinja2 templates
- add basic jinja2 environment to lando

### CODE REVIEW

File: src/lando/jinja2.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from django.templatetags.static import static
```
Comment (cgsheeh) [inline src/lando/jinja2.py:1]: What else is going to go in this module? Could we just put this in `settings.py`?

**Review by cgsheeh — CHANGES_REQUESTED** [review]
Seems out of place to add this without including any templates.

**Comment by zzzeid** [comment]
> Seems out of place to add this without including any templates.

Templates are already in the base branch, and are moved in PR #21, and will be properly ported in a different PR, since they all depend on various functionality that is not available at this time.

File: src/lando/jinja2.py (hunk 1 above)
Comment (zzzeid) [reply src/lando/jinja2.py:1]: > What else is going to go in this module? Could we just put this in `settings.py`?

This is the typical way of adding a Jinja environment. Basically all the filters and custom functionality we added in the legacy repo will eventually go in here. This doesn't belong in the settings module.

**Review by cgsheeh — APPROVED** [review]

**Pull request merged**